projects/packages/my-jetpack/_inc/data/products/use-products-by-ownership.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react';
+import { REST_API_SITE_PRODUCTS_OWNERSHIP_ENDPOINT } from '../constants';
+import { QUERY_PRODUCT_BY_OWNERSHIP_KEY } from '../constants';
+import useSimpleQuery from '../use-simple-query';
+import type { WP_Error } from '../types';
+import type { QueryObserverResult } from '@tanstack/react-query';
+
+// Create query to fetch new product data from the server
+const useFetchProductsByOwnership = () => {
+	const queryResult = useSimpleQuery<
+		Record< 'ownedProducts' | 'unownedProducts', JetpackModule[] >
+	>( {
+		name: `${ QUERY_PRODUCT_BY_OWNERSHIP_KEY }`,
+		query: {
+			path: REST_API_SITE_PRODUCTS_OWNERSHIP_ENDPOINT,
+		},
+	} );
+
+	return queryResult;
+};
+
+// Fetch the product data from the server
+const refetchProduct = async (
```

Comment:
Should we update the function name here? It's more like `refetchProductOwnershipData`. Do we actually need that explicit function? seems like it doesn't add any value over just calling `refetch()` (we don't use result data from there).

Reply:
Good point, thanks!